./util/docker-dev/run.sh gives error: "sudo: no tty present and no askpass program specified"

On macOS 10.15, I get:

 51$ $ ./util/docker-dev/run.sh
[  2019-10-28 17:37:47  INFO  ]  Creating docker instance for dev server. Instance name: wptd-dev-instance
[  2019-10-28 17:37:47  WARN  ]  Found existing docker instance wptd-dev-instance. Remove?
Confirm (y/N): y
[  2019-10-28 17:37:48  INFO  ]  Stopping wptd-dev-instance...
wptd-dev-instance
[  2019-10-28 17:37:49  INFO  ]  wptd-dev-instance stopped.
[  2019-10-28 17:37:49  INFO  ]  Removing wptd-dev-instance...
wptd-dev-instance
[  2019-10-28 17:37:49  INFO  ]  wptd-dev-instance removed.
[  2019-10-28 17:37:49  INFO  ]  Starting docker instance wptd-dev-instance...
416826f8d40a7f2f121517c3242f4be0e82dc0d168dcbf87f0907684417e9c64
[  2019-10-28 17:37:50  INFO  ]  Setting up local user
groupadd: GID '20' already exists
[  2019-10-28 17:37:51  INFO  ]  Ensuring the home directory is owned by the user...
[  2019-10-28 17:37:52  INFO  ]  Instance wptd-dev-instance started.
[  2019-10-28 17:37:52  INFO  ]  Updating system/packages...
sudo apt-get -qq update
sudo: no tty present and no askpass program specified
make: *** [apt_update] Error 1
Makefile:185: recipe for target 'apt_update' failed

--- a/util/commands.sh
+++ b/util/commands.sh
@@ -10,7 +10,7 @@ function wptd_useradd() {
   docker exec -u 0:0 "${DOCKER_INSTANCE}" groupadd -g $(id -g $USER) user || [ $? == 4 ]
   # Add user to audio & video groups to ensure Chrome can use sandbox.
   docker exec -u 0:0 "${DOCKER_INSTANCE}" useradd -u $(id -u $USER) -g $(id -g $USER) -G audio,video user
-  docker exec -u 0:0 "${DOCKER_INSTANCE}" sh -c 'echo "%user ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers'
+  docker exec -u 0:0 "${DOCKER_INSTANCE}" sh -c 'echo "user ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers'
 }
 function wptd_exec() {
   docker exec -u $(id -u $USER) "${DOCKER_INSTANCE}" sh -c "$*"

